next/api/resources/retired.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, send_file
```

Comment:
Yep. Unfortunately, it still has to be a separate resource, thanks to the way Flask-Restful works.